storage/postgres.go

```diff
@@ -16,3 +19,28 @@ type DBConfig struct {
 	Password string
 	DBName   string
 }
+
+const pgStr = "host=%v port=%v user=%v password=%v dbname=%v sslmode=disable"
+const DBDriverName = "postgres"
+
+//DB connector
+func ConnectToDB(pg *DBConfig) (*sql.DB, error) {
```

Comment:
Comment is a complete sentence that begins with the name of the element it describes 
https://blog.golang.org/godoc

Reply:
Fixed it